app/models/invitation.rb

```diff
@@ -13,6 +13,8 @@ class InvitationAlreadyUsed < StandardError
   belongs_to :invitable, polymorphic: true
   belongs_to :canceller, class_name: User
 
+  update_counter_cache :invitable, :invitations_count
```

Comment:
Hey @robguthrie do you have a strong sense of when this `update_counter_cache` fires? Is it after saves? Only saves which change the relevant information? Only after create / delete, and not update?

Also, should this be `update_counter_cache :group, :invitations_count`?